docs/source/markdown/podman-ps.1.md

```diff
@@ -32,12 +32,18 @@ all the containers information.  By default it lists:
 
 **--all**, **-a**
 
-Show all the containers, default is only running containers
+Show all the containers including those created by other container technologies such as Buildah and CRI-O.  The default displays only running containers maintained by Podman.  External containers will have a 'storage' status.
```

Comment:
I'd really like this to read " including non-running ones and those created..." but it is really not worth another CI run for that. Maybe a followup CI docs pr.

Reply:
Can we undo this change until we change the default?